scripts/differential_check.py

```
@@ -0,0 +1,260 @@
+    if used != str(worktree / "dascore"):
+        msg = f"expected the dascore in {worktree}, imported {used}"
+        raise RuntimeError(msg)
+
+
+def main(ref: str) -> int:
+    """Compare the working tree against a git ref."""
+    repo = Path(__file__).resolve().parent.parent
+    with tempfile.TemporaryDirectory() as temp:
+        temp = Path(temp)
+        worktree = temp / "baseline"
+        subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)